pkg/account/delete/delete.go

```diff
@@ -60,55 +67,116 @@ func (a Account) String() string {
 	return fmt.Sprintf("%q (%s)", a.Name, a.UUID)
 }
 
-// AccountResources removes all given Resources from the given Account
+// DeleteAccountResources removes all given Resources from the given Account
 // Returns an error if any resource fails to be deleted, but attempts to delete as many resources as possible and only returns an error at the end.
-func AccountResources(ctx context.Context, account Account, resourcesToDelete Resources) error {
+func DeleteAccountResources(ctx context.Context, account Account, resourcesToDelete Resources) error {
```

Comment:
Functions should not start with the package name, because it would be redundant `delete.Delete...`

Reply:
I agree - but would change the package structure. I've opened a `tech-time` about this, so maybe we can come up with something better.